docs/content/install.sh

```diff
@@ -9,9 +9,23 @@ if [ -n "$1" ] && [ "$1" != "beta" ]; then
 fi
 
 if [ -n "$1" ]; then
-    install_beta="yes"
+    install_beta="beta "
 fi
 
+#check installed version of rclone to determine if update is necessary
+version=`rclone --version | head -n 1`
+if [ -z "${install_beta}" ]; then
+    current_version=`curl https://downloads.rclone.org/version.txt`
+else
+    current_version=`curl https://beta.rclone.org/version.txt`
+fi
+
+if [ "$version" = "$current_version" ]; then
+    echo && echo "The latest ${install_beta}version of rclone is already installed" && echo
+    exit 1
```

Comment:
Maybe change exit code to something else. 
Therefore users can know if they has the newest version installed just by checking exit code.
It will make scripting easier. 
0 - standard exit code
1 - error exit code
2 - up to date

Reply:
Ok, I'll also put similar description of error codes at the beginning of the script, so it will be clear what the they mean.